docs/markdown/Syntax.md

```diff
@@ -706,7 +706,7 @@ DECIMAL_NUMBER: /[1-9][0-9]*/
 dictionary_literal: "{" [key_value_list] "}"
 equality_expression: relational_expression | (equality_expression equality_operator relational_expression)
 equality_operator: "==" | "!="
-expression: assignment_expression
+expression: assignment_statement
 expression_list: expression ("," expression)*
```

Comment:
This still doesn't seem right. Assignment is a statement, but an expression isn't assignment

Reply:
I know ... could really think of a way to resolve this

Reply:
And this should be:
`expression: conditional_expression | (logical_or_expression assignment_operator assignment_expression)`